src/main/kotlin/com/z8dn/plugins/a2pt/nodes/ProjectFileGroupNode.kt

```
@@ -88,4 +94,36 @@ class ProjectFileGroupNode(
+                "AllIcons.General.GearPlain" -> AllIcons.General.GearPlain
+                "AllIcons.Actions.ListFiles" -> AllIcons.Actions.ListFiles
+                "AllIcons.Actions.GroupBy" -> AllIcons.Actions.GroupBy
+                "AllIcons.Actions.Copy" -> AllIcons.Actions.Copy
+                "AllIcons.Actions.Edit" -> AllIcons.Actions.Edit
+                "AllIcons.FileTypes.Config" -> AllIcons.FileTypes.Config
+                "AllIcons.FileTypes.Text" -> AllIcons.FileTypes.Text
+                "AllIcons.FileTypes.Properties" -> AllIcons.FileTypes.Properties
+                "AllIcons.FileTypes.Archive" -> AllIcons.FileTypes.Archive
+                else -> AllIcons.Nodes.Folder // Default fallback
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)